.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
```

Comment:
I don't think that husky is actually working when I pull down the repo and try it locally. I think we're missing either a `postinstall` hook (like we have in Dragon), or a `prepare` hook as mentioned in the husky docs.

Reply:
Uh, I'll check it later today 👀

Reply:
Did you want to go ahead and get this in and then handle the husky stuff in a subsequent PR?

Reply:
No, let me try to get it (or you can try it yourself too). I need to leave now, will come back at ~21:30 UTC.

Reply:
No, let me try to get it (or you can try it yourself too). I need to leave now, will come back at ~21:30 UTC.

Reply:
All fixed!

- husky: https://github.com/remoteoss/json-schema-form/pull/1/commits/a115843bade02262fe19d9540c6947b96455c6b4
- vercel build link: it [did run in the first commit](https://github.com/remoteoss/json-schema-form/actions/runs/4994138803) but it failed and then I fixed on a following commit
- bonus: I noticed ESLint rules were not working properly (eg unused vars) so I've added the missing plugins/config https://github.com/remoteoss/json-schema-form/pull/1/commits/3c23626b00abf564439f4bbee603c4c5663aaeb6

Reply:
Ready for a final review ⚡

Reply:
Relooking over this, will test husky and eslint now

Reply:
retesting here worked well, with the same functionality we have internally. Linting and formatting works as I expect to run on commits. Awesome. lets go.

Reply:
Cool! Let's go? I finished rebasing with main to add the co-authors, could you re-approve?